helpers/messages/crosspostMessage.ts

```diff
@@ -3,7 +3,7 @@ import { Message } from "../../transformers/message.ts";
 import { DiscordMessage } from "../../types/discord.ts";
 
 /** Crosspost a message in a News Channel to following channels. */
-export async function publishMessage(bot: Bot, channelId: bigint, messageId: bigint): Promise<Message> {
+export async function crosspostMessage(bot: Bot, channelId: bigint, messageId: bigint): Promise<Message> {
```

Comment:
i would keep an alias for this as imo this is another case where publish is more intuitive as this is the name discord calls it. When you click the button it asks if u want to publish it, so as you code your brain will first search for `publish` and autocomplete will show it. The name change is good and accurate to the api name, but id recommend a alias for it also.

Reply:
Added alias in latest commit.